include/radio.hpp

```diff
@@ -0,0 +1,21 @@
+#include "config.hpp"
+
+#ifndef RADIO_H
+#define RADIO_H
+
+namespace bzzz
+{   
+    /**
+     * @brief Read the receiver data sent from Pi
+     */
+    void readPiData(void);
+
+    /**
+     * @brief reformat receiver values
+     */
+    void readRadioData(int &radioThrottle, int &radioRoll, int &radioPitch, int &radioYaw, int &radioSwitchC,
```

Comment:
In general it's not good practice to have a function with so many arguments. The question is how we plan to read the values from the RC. We could have a function like

```c++
void switches(bool* switchesValues);
void referenceQuaternion(float* refQuaternion);
void trimmerValues(int* trimmers);
```